### DISCUSSION TITLE
Is it possible to manage date values expressed in languages/locale other than English?

### DISCUSSION THREAD

**Discussion by aborruso** [johnkerl/miller#476] [Q&A]
Hi,
I live in Italy and date values are often written in Italian.

I example the Italian date `3 Maggio 2017` is `2017-05-03` ISO date.  To convert the former value to YYYY-MM-DD usually I use the great [dateutils](https://github.com/hroptatyr/dateutils) (or Python):

```
echo "3 Maggio 2017" | dconv --from-locale it_IT  -i '%d %B %Y'
```

Is there a DSL way inside Miller (without using `system` funtion) to manage date values expressed in languages/locale other than English?

Thank you

**Comment by johnkerl** [comment]
This will be a great opportunity for the Go port -- datetime support in Go is far better and more portable than in the C library.

**Reply by johnkerl** [reply]
https://github.com/johnkerl/miller/issues/477

**Reply by trantor** [reply]
Just as a reference @aborruso @johnkerl , although not strictly related to miller, I had a problem with locale-dependent dates in jq as well https://github.com/stedolan/jq/issues/1740
Still unresolved as well

**Comment by masgo** [comment]
There is a `system` function in miller. Performance-wise it's obviously not that good, but you can use it to solve "strange" problems by using an external program, `date` in your case. 

Here is an example from one of my scripts. I use it to determine the file-date for a table with filePaths. 
`$timestamp=system("stat -c %Y ".$filePath)`
